src/platform/nrfconnect/Reboot.cpp

```diff
@@ -21,14 +21,14 @@
 
 #include <zephyr/sys/reboot.h>
 
-#ifndef CONFIG_ARCH_POSIX
+#if !(defined(CONFIG_ARCH_POSIX) || defined(CONFIG_BOARD_NRF54H20DK_NRF54H20_CPUAPP))
 #include <hal/nrf_power.h>
 #endif
 
 namespace chip {
 namespace DeviceLayer {
 
-#ifdef CONFIG_ARCH_POSIX
+#if defined(CONFIG_ARCH_POSIX) || defined(CONFIG_BOARD_NRF54H20DK_NRF54H20_CPUAPP)
```

Comment:
TBH this is quite ugly workaround :) Could we at least use the SoC name and not the DK in the source code? This will not apply if someone would create own device based on our SoC.

Reply:
Right, I will change it.